src/pages/home/report/ContextMenu/BaseReportActionContextMenu.js

```diff
@@ -37,7 +37,14 @@ class BaseReportActionContextMenu extends React.Component {
     }
 
     render() {
-        const shouldShowFilter = contextAction => contextAction.shouldShow(this.props.type, this.props.reportAction, this.props.isArchivedRoom, this.props.betas, this.props.anchor);
+        const shouldShowFilter = contextAction => contextAction.shouldShow(
+            this.props.type,
+            this.props.reportAction,
+            this.props.isArchivedRoom,
+            this.props.betas,
+            this.props.anchor,
+            this.props.isChronosReport,
```

Comment:
Will you please add `isChronosReport` and `isArchivedRoom` in `propTypes` and `defaultProps`?

Reply:
@sobitneupane Added , Thanks